wallet/src/main/resources/postgresql/wallet/migration/V4__wallet_db_pg_baseline.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE wallet_accounts (hd_purpose INTEGER NOT NULL,xpub TEXT NOT NULL,coin INTEGER NOT NULL,account_index INTEGER NOT NULL,constraint pk_account primary key(hd_purpose,coin,account_index));
```

Comment:
How do we decide which migrations to run?

Reply:
Based on which database driver is defined in the config